README.md

```diff
@@ -371,6 +371,18 @@ result = client.execute_function(full_func_name, {"s": "some_string"})
 assert result.value == "some_string"
 ```
 
+##### Function execution arguments configuration
```

Comment:
This is what it looks like:

<img width="962" alt="image" src="https://github.com/user-attachments/assets/82645b95-d646-45e7-a0ce-dc5a9eea4285">

It's a bit challenging to tell what environment variables are available. I'd use a table.

Reply:
Updated! 
<img width="1103" alt="image" src="https://github.com/user-attachments/assets/6e0e1beb-ac47-421a-83ff-f8cf16f85a88">